terraform/infrastructure/etc/qa.tfvars

```diff
@@ -3,3 +3,4 @@ account_name = "qa"
 domain                = "qa.record-locator.national.nhs.uk"
 public_domain         = "internal-qa.api.service.nhs.uk"
 public_sandbox_domain = "internal-qa-sandbox.api.service.nhs.uk"
+log_retention_period  = 90
```

Comment:
as with dev, seems like a long period for a qa env. Happy to review over time though.